README.md

```diff
@@ -1,70 +1,74 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Todo Application
 
-## Available Scripts
+> This is an application that stores lists of books
 
-In the project directory, you can run:
 
-### `npm start`
+## Live Demo
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[Live demo]()
+## Built With
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- HTML
+- CSS
+- Javascript
+- React
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Prerequisites
 
-### `npm run build`
+- Node
+- NPM ( Node Package Manager )
+- Git and Github Account
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Setup
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Run git clone on this project at the desired directory:
+  ```
+  git clone https://github.com/Fadahunsiseyi/react-book-store.git
+  ```
+  ```
+  cd react-todo-app
+  ```
```

Comment:
- [ ] OPTIONAL Please, consider fixing this command, because it should be `cd react-book-store`